From a466f5dda467c815e46c22c18a53a10d4b849474 Mon Sep 17 00:00:00 2001
From: Lu Jialin <lujialin4@huawei.com>
Date: Fri, 16 Jul 2021 12:20:46 +0800
Subject: [PATCH] fs/filescontrol.c: fix warning:large integer implicitly
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: a466f5dda467c815e46c22c18a53a10d4b849474

 truncated to unsigned type

hulk inclusion
category: bugfix
bugzilla: 50779
CVE: NA

--------

page_counter_set_max(struct page_counter *counter, unsigned long nr_pages)
the nr_pages is unsigned long, therefore change FILES_MAX to ULONG_MAX

Signed-off-by: Lu Jialin <lujialin4@huawei.com>
Reviewed-by: Xiu Jianfeng <xiujianfeng@huawei.com>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 fs/filescontrol.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fs/filescontrol.c b/fs/filescontrol.c
index 41abe29fc0f8..4ad500f40025 100644
--- a/fs/filescontrol.c
+++ b/fs/filescontrol.c
@@ -27,7 +27,7 @@
 #include <linux/sched/signal.h>
 #include <linux/module.h>
 
-#define FILES_MAX ULLONG_MAX
+#define FILES_MAX ULONG_MAX
 #define FILES_MAX_STR "max"
 
 static bool no_acct;
-- 
2.26.2

